sct-commons/src/main/java/org/lfenergy/compas/sct/commons/DoService.java

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: 2022 RTE FRANCE
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2023 RTE FRANCE
```
Same thing for newly created classes.

Reply:
done